### ISSUE TITLE
Calculating Accuracy using History Object of Keras fit() method

### ISSUE THREAD

**Issue by Suhird** [matterport/Mask_RCNN#1172]
Hi
I have a sample data set on which I am running maskRCNN. I am creating a model model = modellib.MaskRCNN(mode='training', config=config, model_dir=MODEL_DIR) and training model using 
model.train(dataset_train, dataset_val, 
            learning_rate=config.LEARNING_RATE, 
            epochs=NUM_EPOCHS, 
            layers='all',
            augmentation = augmentation)
I want to somehow plot the accuracy after the training is done. Is there  a way i can get the History object as we get it in Keras fit() method or can someone suggest me how to use Tensorboard for this trained model?

Regards
Suhird

**Comment by aliffarisqi** [comment]
i have same question bro, how to calculate it ?
